frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -0,0 +1,164 @@
+      if (usedPool.indexOf(pool?.metadata.name) === -1 && pool?.status?.phase === 'Ready') {
+        res.push(
+          <DropdownItem
+            key={pool.metadata.uid}
+            component="button"
+            id={pool?.metadata?.name}
+            onClick={handleDropdownChange}
+            description={`Replica ${pool?.spec?.replicated?.size}, ${
+              pool?.spec?.compressionMode === 'none' || pool?.spec?.compressionMode === ''
+                ? 'no compression'
+                : 'with compression'
+            }`}
+          >
+            {pool?.metadata?.name}
+          </DropdownItem>,
+        );
+      }
+      return res;
+    },
+    [
+      <DropdownItem
+        key="first-item"
+        component="button"
+        onClick={() =>
+          storagePoolModal({
+            cephClusterObj,
+            onPoolCreation,
+          })
+        }
+      >
+        Create New Pool
+      </DropdownItem>,
+      <DropdownSeparator key="separator" />,
+    ],
+  );
+
+  if (cephClusterObj[0]?.metadata.name === CEPH_INTERNAL_CR_NAME) {
+    return (
+      <>
+        {!poolDataLoadError && cephClusterObj && (
+          <div className="form-group">
+            <label className="co-required" htmlFor="ocs-storage-pool">
+              Storage Pool
+            </label>
+            <Dropdown
+              className="dropdown dropdown--full-width"
+              toggle={
+                <DropdownToggle
+                  id="toggle-id"
+                  onToggle={() => setOpen(!isOpen)}
+                  toggleIndicator={CaretDownIcon}
+                >
+                  {poolName || 'Select a Pool'}
+                </DropdownToggle>
+              }
+              isOpen={isOpen}
+              dropdownItems={poolDropdownItems}
+              onSelect={() => setOpen(false)}
+              id="ocs-storage-pool"
+            />
+            <span className="help-block">Ceph pool into which volume data shall be stored</span>
+          </div>
+        )}
+        {(poolDataLoadError || loadError) && (
+          <Alert
+            className="co-alert"
+            variant="danger"
+            title="Error retrieving Parameters"
+            isInline
+          />
+        )}
+      </>
+    );
+  }
+  if (cephClusterObj[0]?.metadata.name === CEPH_INTERNAL_CR_NAME) {
+    return (
+      <div className="form-group">
+        <label className="co-required" htmlFor="ocs-storage-pool">
+          Storage Pool
+        </label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          onChange={(e) => setPoolName(e.currentTarget.value)}
+          placeholder="my-storage-pool"
+          aria-describedby="pool-name-help"
+          id="pool-name"
+          name="newPoolName"
+          required
+        />
+        <span className="help-block">Ceph pool into which volume data shall be stored</span>
+      </div>
+    );
+  }
+  return <>{loaded || poolDataLoaded || <LoadingInline />}</>;
```

Comment:
Shouldn't this be `{(loaded || poolDataLoaded) && <LoadingInline />}`

Reply:
ack